src/main/scala/fi/oph/koski/schema/VapaaSivistystyö.scala

```diff
@@ -68,6 +53,20 @@ case class VapaanSivistystyönOpiskeluoikeudenLisätiedot(
 
 trait VapaanSivistystyönPäätasonSuoritus extends KoskeenTallennettavaPäätasonSuoritus with Toimipisteellinen with Suorituskielellinen with Todistus with Arvioinniton
 
+@Title("Oppivelvollisille suunnatun vapaan sivistystyön opiskeluoikeusjakso")
+@OnlyWhen("../../../suoritukset/0/tyyppi/koodiarvo", "vstoppivelvollisillesuunnattukoulutus")
+@OnlyWhen("../../../suoritukset/0/tyyppi/koodiarvo", "vstmaahanmuuttajienkotoutumiskoulutus")
+@OnlyWhen("../../../suoritukset/0/tyyppi/koodiarvo", "vstlukutaitokoulutus")
+case class OppivelvollisilleSuunnattuVapaanSivistystyönOpiskeluoikeusjakso(
```

Comment:
Varmistelen, että tsekkasitko tuotannosta, että ei ole opiskeluoikeuksissa tai niiden historiassa tämän seuraksena deserialisoitumatonta dataa?

Reply:
Ei näitä olis pitänytkään olla, koska kyllähän näille sittenkin oli validaatio (jonka otin päällekkäisenä validaationa kuitenkin pois). Mut tarkistin kyl ja ei ole tuotannon kannassa deserialisoitumatonta kamaa.